core/src/main/java/cucumber/runtime/arquillian/client/CucumberArchiveProcessor.java

```diff
@@ -355,4 +329,38 @@ private static String featureName(final URL url) {
         // fallback
         return Math.abs(url.hashCode()) + Features.EXTENSION;
     }
+
+    private static boolean isCukeSpaceTest(final TestClass testClass, final ClassLoader loader) {
+        Class<? extends Annotation> runWithType = null;
+        try {
+            runWithType = (Class<? extends Annotation>) loader.loadClass("org.junit.runner.RunWith");
+
+        } catch (final ClassNotFoundException error) {
+            // no-op
+        } catch (final NoClassDefFoundError e) {
+            // no-op
+        }
+
+        if (runWithType == null) {
+            return false;
+        }
+        final Annotation runWith = testClass.getAnnotation(runWithType);
+        if (runWith == null) {
+            return false;
+        }
+
+        try {
+            final Class<?> runner = Class.class.cast(runWithType.getMethod("value").invoke(runWith));
+            if ((!ArquillianCucumber.class.equals(runner) && !CukeSpace.class.equals(runner))) {

```

Comment:
How is it possible that you have seen some children relying on default ?? 
The current code doesn't allow this (equal on class type) ??!!  :)
What is the reason to extends the runner and use the default enricher ?
I think the runner annotation is provided and must be used as it is, if it is extended then it means that you want to wrap the runner and so you have to change something else (e.g. create your own enricher)
Anyway What do you mean with having the enrichment configurable ?
It could be interesting  in general .. but my point of view on the annotation is always  the same :p
For example it could be useful to use an helper class to enrich a given WebArchive with the cucumber stuff.
This will give more chance to customize cucumber runner without replicating the enrichment code.